jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/BrowserOpen.java

```diff
@@ -0,0 +1,34 @@
+package io.github.epam.html.tests.elements.common;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.settings.WebSettings.logger;
+
+public class BrowserOpen {
+
+    private WebDriver driver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'driver' could be replaced by a local variable.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4665121)